bin/alks-iam-createtrustrole

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env node
+'use strict';
+
+process.title = 'ALKS';
+
+var program   = require('commander'),
+    _         = require('underscore'),
+    clc       = require('cli-color'),
+    alks      = require('alks-node'),
+    Iam       = require('../lib/iam'),
+    utils     = require('../lib/utils'),
+    Developer = require('../lib/developer'),
+    config    = require('../package.json');
+
+var logger       = 'iam-createtrustrole',
+    roleNameDesc = 'alphanumeric including @+=._-',
+    trustArnDesc = 'arn:aws|aws-us-gov:iam::d{12}:role/TestRole';
+
+program
+    .version(config.version)
+    .description('creates a new IAM Trust role')
+    .option('-n, --rolename [rolename]', 'the name of the role, ' + roleNameDesc)
+    .option('-t, --roletype [roletype]', 'the role type, to see available roles: Cross Account')
```

Comment:
What does this mean? "To see available roles: Cross Account"?

Reply:
Wait, so it only supports `Cross Account` as the role type? Why not just hardcode that value and remove the option all together?